webapp/components/file_preview.jsx

```diff
@@ -39,7 +39,7 @@ export default class FilePreview extends React.Component {
                 previewImage = (
                     <img
                         className='file-preview__image'
-                        src={getFileUrl(info.id)}
+                        src={getFilePreviewUrl(info.id)}
```

Comment:
Was using preview url before because thumbnail url had wrong aspect ratio. Looks like that's been fixed on the server, so using thumbnail url now.